packages/firestore/test/unit/model/mutation.test.ts

```diff
@@ -478,21 +481,34 @@ describe('Mutation', () => {
       /*transformResults=*/ null
     );
     const transformResult = new MutationResult(version(7), []);
+    const docV7Unknown = unknownDoc('collection/key', 7);
+    const docV7Deleted = deletedDoc('collection/key', 7);
+    const docV7Committed = doc(
+      'collection/key',
+      7,
+      {},
+      { hasCommittedMutations: true }
+    );
 
-    assertVersionTransitions(set, docV3, mutationResult, docV3);
-    assertVersionTransitions(set, deletedV3, mutationResult, docV0);
-    assertVersionTransitions(set, null, mutationResult, docV0);
+    assertVersionTransitions(set, docV3, mutationResult, docV7Committed);
```

Comment:
This test essentially implements the version transitions in that big comment on top of Mutation. Could you update the comment? Alternatively, if you didn't find the table helpful, maybe we we should just delete it?

Reply:
The test invokes `applyToRemoteDocument`, which has changed a lot and hence the changes to the test. The table in the comment is for `applyToLocalView`, which should still be the same.